examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -436,19 +432,19 @@ var _ = ginkgo.Describe("[Tx Processing]", func() {
 			// read: 2 keys reads, 1 had 0 chunks
 			// allocate: 1 key created with 1 chunk
 			// write: 2 keys modified (new + old)
-			transferTxConsumed := fees.Dimensions{191, 7, 12, 25, 26}
+			transferTxConsumed := fees.Dimensions{187, 7, 12, 25, 26}
```

Comment:
Removed https://github.com/ava-labs/hypersdk/blob/203e78b7c65a8b7c875fa4941f974292ce989f52/chain/transaction.go#L76C9-L76C17 which returns an `int` which is 4 bytes 

Which also resulted in updating the numbers below this line for the rest of the file as well